apps/website/src/content/docs/linkaction.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: LinkAction
+description: Helps create accessible block links or buttons.
+thumbnail: Anchor
+group: utilities
+---
+
+`LinkAction` is a utility component that makes it easier to create accessible [block links](https://adrianroselli.com/2020/02/block-links-cards-clickable-regions-etc.html) (or buttons).
+
+<LiveExample src='LinkAction.main.tsx'>
+  <AllExamples.LinkActionMainExample client:load />
+</LiveExample>
+
+This component is useful in a wide range of situations when you need to increase the clickable area of an element (e.g. a card, a list item, etc.) without compromising semantics. This means you can keep the link text concise and still make the entire area clickable, even if it includes other elements or extra padding.
+
+`LinkAction` is used internally in [`Tile`](tile#actionable) and [`ListItem`](list#with-linkaction) components, and is also available for use in external components.
+
+## Usage
+
+This component consists of two parts:
+
+1. `LinkBox`: A wrapper component that makes the entire area clickable.
```

Comment:
Can `LinkAction` be used without a `LinkBox`? If so, should that be documented?

Reply:
technically yes if the container uses `position: relative`, but i would feel more comfortable if users added `LinkBox` anyway

the only exception is within our other components like `Tile` and `ListItem`, but i'm thinking we should have wrappers for all of those (e.g. `ListItem.Action`, similar to `Tile.Action`)

Reply:
Technically it can if the container has `position: relative`, but I would feel more comfortable if `LinkBox` was always used. Gives us the ability to add more logic to it in the future